query/src/main/java/io/keen/client/java/QueryType.java

```diff
@@ -0,0 +1,65 @@
+package io.keen.client.java;
+
+/**
+ * QueryType specifies which query to run.
+ *
+ * Created by claireyoung on 6/16/15.
+ */
+public enum QueryType {
+    COUNT_RESOURCE,
```

Comment:
Nit: `QueryType.COUNT_RESOURCE` feels unnecessarily verbose to me. Maybe just `QueryType.COUNT`, i.e. drop the "RESOURCE" portion from them? Doesn't seem like that's adding much value or clarity.